extractors/personal_identifiers/zipcode_extraction/code_snippet_common.md

```diff
@@ -3,17 +3,198 @@ import re
 import spacy
 from typing import List, Tuple
 
-def zipcode_extraction(text: str, extraction_keyword: str, country_id: str) -> List[Tuple[str, int]]:
+
+zip_regex_lookup = {
```

Comment:
Maybe we should move `zip_regex_lookup` to the bottom? It was at the bottom before, now it is a bit tedious to get to the actual implementation. We also do it like this with the refinery code.

Reply:
moved below code snipped